epub33/rs/index.html

```diff
@@ -1165,13 +1165,11 @@ <h3>OCF Abstract Container</h3>
 				<section id="sec-container-iri">
 					<h4>URL of the Root Directory</h4>
 
-					<p>Reading Systems MUST assign a URL [[URL]] to the <a>Root Directory</a> of the <a>OCF Abstract Container</a>. This URL is called the <a data-cite="epub-33#dfn-container-root-url">container root URL</a>. The URL itself is implementation specific, but:</p>
+					<p>Reading Systems MUST assign a URL [[URL]] to the <a>Root Directory</a> of the <a>OCF Abstract Container</a>. This URL is called the <a data-cite="epub-33#dfn-container-root-url">container root URL</a>. It is implementation specific, but the implementation MUST have the following properties:</p>
 
 					<ul>
-						<li>The result of <a data-cite="url#concept-url-parser">parsing</a> [[URL]] "<code>/</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a> MUST be the <a>container root URL</a>.</li>
-						<li>The result of <a data-cite="url#concept-url-parser">parsing</a> [[URL]] "<code>..</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a> MUST be the <a>container root URL</a>.</li>
-						<li>The <a data-cite="url#origin">origin</a> of the <a>container root URL</a> MUST be unique for each EPUB Publication instance.</li>
-						<li>The <a data-cite="url#origin">origins</a> of the <a data-cite="epub-33#dfn-content-url">content URLs</a> of all the files in the <a>OCF Abstract Container</a> MUST be identical.</li>	
+						<li>The result of <a data-cite="url#concept-url-parser">parsing</a> [[URL]] "<code>/</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a> is the <a>container root URL</a>.</li>
+						<li>The result of <a data-cite="url#concept-url-parser">parsing</a> [[URL]] "<code>..</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a> is the <a>container root URL</a>.</li>
```

Comment:
It seems the "origin MUST be unique per instance" criteria (3d bullet) has been unintentionally removed?

Reply:
Ouch

Reply:
Ouch... Thanks. Added that one back.